File: app/src/main/java/ethiopia/covid/android/ui/adapter/OnImageItemClicked.java
package ethiopia.covid.android.ui.adapter;

import android.widget.ImageView;

import java.util.List;

/**
 * Created by BrookMG on 10/22/2019 in ethiopia.covid.android.ui.adapters
 * inside the project CoVidEt .
 */
public interface OnImageItemClicked {
    void OnImageClicked(ImageView imageView , List<String> clickedImageUri);
}

File: app/src/main/java/ethiopia/covid/android/ui/adapter/OnImageItemClicked.kt
package ethiopia.covid.android.ui.adapter

import android.widget.ImageView

/**
 * Created by BrookMG on 10/22/2019 in ethiopia.covid.android.ui.adapters
 * inside the project CoVidEt .
 */
interface OnImageItemClicked {
    fun onImageClicked(imageView: ImageView?, clickedImageUri: List<String?>?)
}